src/qa4sm_reader/plotter.py

```diff
@@ -452,15 +452,17 @@ def _barplot_definition(self,
         figwidth = globals.boxplot_width * (len(df.columns) + 1)
         # otherwise it's too narrow
         figsize = [figwidth, globals.boxplot_height]
-        fig, ax = plm.barplot(df=df, figsize=figsize, label=label)
+        fig, ax = plm.barplot(df=df,
+                              figsize=figsize,
+                              label='# of validation errors')
         if not Var:
             # when we only need reference dataset from variables (i.e. is the same):
             for Var in self.img._iter_vars(type="metric",
                                            filter_parms={"metric": metric}):
                 Var = Var
                 break
 
-        title = self.create_title(Var, type=type)
+        title = globals.status_title
```

Comment:
Can this be wrapped in the `self.create_title` function?

Reply:
done.